src/components/bullet.js

```diff
@@ -181,7 +181,7 @@ AFRAME.registerComponent('bullet', {
         }
       } else {
         // @hack Any better way to get the head position ?
-        var head = this.el.sceneEl.camera.el.components['look-controls'].dolly.position;
```

Comment:
`dolly` no longer exists in `look-controls`, but taking out the `dolly` had the effect of making all enemy bullets aim at the avatar's feet, rather than the headset. I might be missing some way to get the headset position, or we can hack it by manually adding the 1.6 meter height correction.